lexos/models/stats_model.py

```diff
@@ -133,18 +134,32 @@ def _get_document_statistics_dataframe(self) -> pd.DataFrame:
         file_stats[f"Distinct number of {self._token_type_str}"] = \
             self._active_doc_term_matrix.ne(0).sum(axis=1).values
         # Find average number of appearance of tokens.
-        file_stats[f"Average number of {self._token_type_str}"] = \
-            file_stats[f"Total number of {self._token_type_str}"] / \
-            file_stats[f"Distinct number of {self._token_type_str}"]
+        file_stats[f"Vocabulary Density"] = \
+            file_stats[f"Distinct number of {self._token_type_str}"]/\
+            file_stats[f"Total number of {self._token_type_str}"]
+
 
         return file_stats
 
+    def __get_quartile__(self, index: float, arr: np.array) -> float:
```

Comment:
Even if it is a private function, it should be properly documented. This function might be a `staticmethod`.

Reply:
Also this function doesn't really get the `index` quartile of `arr`, I would recommend it actually gets the input quartile of the array (also, are you assuming the `arr` is sorted?)